tests/integration_cases/test_raft.rs

```diff
@@ -4364,3 +4291,284 @@ fn test_conf_change_check_before_campaign() {
     }
     assert_eq!(nt.peers[&1].state, StateRole::Candidate);
 }
+
+fn prepare_request_snapshot() -> (Network, Snapshot) {
+    /************************
```

Comment:
Please uncomment or clean up.

Reply:
I guess you review the code before the `fix test` commit.

Reply:
Oh, I missed it.